doc/articles/features/using-skia-rendering.md

```diff
@@ -0,0 +1,150 @@
+
+## Using Skia rendering for existing apps
+
+To enable Skia rendering in your Uno Platform project, you must opt in using MSBuild properties and features.
+
+### Using Skia Desktop
+
+On **macOS** and **Linux**, Skia rendering is used by default. On **Windows**, Skia is also the default when using the **Desktop target**.  
+However, when using **WinAppSDK**, the **native rendering engine** is always used, regardless of whether Skia or Native is selected.  
+The **MacCatalyst** target has been removed from project templates starting with **Uno SDK 6.0**.
+
+You can find more details in [Using the Skia Desktop](xref:Uno.Skia.Desktop).
+
+### Using Skia Mobile/Wasm
+
+1. To enable Skia rendering on **WebAssembly**, **Android**, and **iOS**, ensure your project is using `Uno.Sdk` 6.0+. For help upgrading, refer to our [migrating guide](xref:Uno.Development.MigratingFromPreviousReleases).
+
+1. Next, enable Skia rendering by adding it to the `<UnoFeatures>` section in your `.csproj` file, like this:
+
+```diff
+<UnoFeatures>
+  <!-- Existing features -->
++  SkiaRenderer;
+</UnoFeatures>
+```
+
+#### Android
+
+No additional setup is required for Skia on Android, but make sure to remove any legacy image loading or custom rendering logic that conflicts with Skia. For example, if you were previously using `UniversalImageLoader`, you can safely remove it:
+
+```diff
+ public Application(IntPtr javaReference, JniHandleOwnership transfer)
+     : base(() => new App(), javaReference, transfer)
+ {
+-    ConfigureUniversalImageLoader();
+ }
+
+-private static void ConfigureUniversalImageLoader()
+-{
+-    var config = new ImageLoaderConfiguration.Builder(Context).Build();
+-    ImageLoader.Instance.Init(config);
+-    ImageSource.DefaultImageLoader = ImageLoader.Instance.LoadImageAsync;
+-}
+```
+
+#### iOS
+
+Replace the default `UIApplication.Main` startup with Skia’s iOS entry point:
+
+```diff
++using Uno.UI.Runtime.Skia.AppleUIKit;
+
+ public static class Program
+ {
+     public static void Main(string[] args)
+     {
+         App.InitializeLogging();
+ 
+-        UIApplication.Main(args, null, typeof(App));
++        var host = new AppleUIKitHost(() => new App());
++        host.Run();
+     }
+ }
+```
+
+#### WebAssembly (Wasm)
+
+For WebAssembly, update the `Program` class to use the `WebAssemblyBrowserHost` provided by the Skia integration:
+
+**Before:**
+
+```csharp
+public class Program
+{
+    private static App? _app;
+
+    public static int Main(string[] args)
+    {
+        App.InitializeLogging();
+
+        Microsoft.UI.Xaml.Application.Start(_ => _app = new App());
+
+        return 0;
+    }
+}
+```
+
+**After:**
+
+```csharp
+using Uno.UI.Runtime.Skia.WebAssembly.Browser;
+
+public class Program
+{
+    private static App? _app;
+
+    public static async Task<int> Main(string[] args)
+    {
+        App.InitializeLogging();
+
+        var host = new WebAssemblyBrowserHost(() => _app = new App());
```

Comment:
This is now outdated, you should use `UnoPlatformBuilder` with `UseWebAssembly()`